### DISCUSSION TITLE
Report Abuse

### DISCUSSION THREAD

**Discussion by LyLme** [community/community#18862] [Code Security]
**Report Abuse: cirosantilli (user)**
https://github.com/cirosantilli

**Reason**

This project is politically sensitive. As a technology sharing website, github should not be used as a political propaganda tool for some people. There is a lot of bloody, violent and fake content inside the project, which has interfered with me and many other users, and made it difficult for us to use github produces a bad experience.

**Comment by Zerotask** [comment]
There is a report option on profile pages
